depobs/worker/tasks.py

```diff
@@ -382,3 +383,54 @@ def fetch_and_save_registry_entries(package_names: Iterable[str]) -> List[Dict]:
         )
     )
     return npm_registry_entries
+
+
+def get_github_advisories(package_name: str) -> List[Dict]:
+
+    github_client = current_app.config["GITHUB_CLIENT"]
+    base_url = github_client["base_url"]
+    github_auth_token = github_client["github_auth_token"]
+
+    headers = {"Authorization": "token " + github_auth_token}
+
+    query = (
+        """
+    {
+        securityVulnerabilities(ecosystem: NPM, first: 100, package: \""""
+        + package_name
+        + """\", orderBy: {field: UPDATED_AT, direction: DESC}) {
+            nodes {
+                advisory {
+                    id, description, permalink, publishedAt, severity, summary, updatedAt, withdrawnAt
+                }
+                package {
+                    name
+                }
+            }
+            pageInfo {
+                endCursor, hasNextPage, hasPreviousPage, startCursor
+            }
+            totalCount
+        }
+    }
+    """
+    )
+
+    response = requests.post(base_url, json={"query": query}, headers=headers)
+    response.raise_for_status()
+    response = response.json()
```

Comment:
mypy is unhappy with `response` being aliased, so we should rename this one and in the following line or inline `response.json()` in the following line